modules/command/common/send-telemetry-command.js

```diff
@@ -107,7 +53,7 @@ class SendTelemetryCommand extends Command {
             data: {
                 message: '',
             },
-            period: 20 * 60 * 1000, // 5 * 60 * 1000
+            period: 1 * 60 * 1000, // 5 * 60 * 1000
```

Comment:
We probably don't want to leave it to run every minute

Reply:
Correct, I will update it